__tests__/api.test.js

```diff
@@ -149,3 +149,68 @@ describe("5. PATCH /api/reviews/:review_id", () => {
       });
   });
 });
+
+describe("2. GET /api/reviews", () => {
+  test("status:200, responds with every review in DESC order", () => {
+    return request(app)
+      .get(`/api/reviews`)
+      .expect(200)
+      .then(({ body }) => {
+        let review = body.review;
+        expect(review.length).toEqual(13);
+        expect(review).toBeSortedBy("created_at", {
+          descending: true,
+        });
+        review.forEach(review => {
+          expect(review).toEqual(
+            expect.objectContaining({
+              review_id: expect.any(Number),
+              title: expect.any(String),
+              category: expect.any(String),
+              designer: expect.any(String),
+              owner: expect.any(String),
+              review_body: expect.any(String),
```

Comment:
we don't want to have review_body on the object sent back, so let's change the model so that it doesn't request it.  (This wasn't written explicitly on the trello so you haven't missed anything).  You'll find out the reason for this in a few weeks :D